src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/LibertyServer.java

```diff
@@ -79,36 +79,8 @@ public void createFiles(JessieModel model) {
 
     @Override
     public void adaptMavenModel(Model pomFile, JessieModel model, boolean mainProject) {
-        String openLibertyVersion = "RELEASE";
-        String openLibertyMavenVersion = "";
-
-        switch (model.getSpecification().getMicroProfileVersion()) {
-
-            case NONE:
-                break;
-            case MP30:
-                openLibertyMavenVersion = "2.2";
-                break;
-            case MP22:
-                openLibertyMavenVersion = "2.2";
-                break;
-            case MP21:
-                openLibertyMavenVersion = "2.2";
-                break;
-            case MP20:
-                openLibertyMavenVersion = "2.2";
-                break;
-            case MP14:
-                openLibertyMavenVersion = "2.0";
-                break;
-            case MP13:
-                openLibertyMavenVersion = "2.0";
-                break;
-            case MP12:
-                openLibertyMavenVersion = "2.0";
-                break;
-            default:
-        }
+        String openLibertyVersion = "[19.0.0.9,)";
```

Comment:
Dependency ranges leads to non-repeatable build of the projects

Reply:
Previously it was `Latest`, which always downloads the latest, whose behavior was not updated by this PR. If someone want to lock down the version after the project is generated, maybe we can add a comment prior to  the version generation in the generated pom.xml.

Reply:
This should be fixed with the new plugin. I verified locally. Let's try again in the test server.